app/src/test/kotlin/com/intive/picover/auth/repository/AuthRepositoryTest.kt

```diff
@@ -127,13 +127,23 @@ class AuthRepositoryTest : ShouldSpec(
 		should("return Profile WHEN updateUserAvatar called") {
 			coEvery { referenceToUserAvatar.downloadUrl.await() } returns userPhoto
 
-			tested.updateUserAvatar(userPhoto) shouldBe profile
+			val result = tested.updateUserAvatar(userPhoto)
+
+			result.isSuccess shouldBe true
+			result.onSuccess {
+				it shouldBe profile
+			}
 		}
 
 		should("return Profile WHEN userProfile called") {
 			coEvery { referenceToUserAvatar.downloadUrl.await() } returns userPhoto
 
-			tested.userProfile() shouldBe profile
+			val result = tested.userProfile()
+
+			result.isSuccess shouldBe true
+			result.onSuccess {
+				it shouldBe profile
+			}
```

Comment:
Please check other methods of `kotlin.Result`, there is a nice candidate to combine these two assertions into one :)

Reply:
Thanks, `shouldBeSuccess` will be appropriate

Reply:
Done